source/plugin/plugin.js

```diff
@@ -149,6 +146,9 @@ Webpack_isomorphic_tools_plugin.prototype.apply = function(compiler)
 			notify_stats(stats, json, plugin.options.verbose)
 		}
 
+		// // assets base path (on disk or on the network, also add support to webpack-dev-server)
+		const assets_base_path = (webpack_configuration.devServer && webpack_configuration.devServer.publicPath) ? webpack_configuration.devServer.publicPath : json.publicPath
+
```

Comment:
When the devServer.publicPath exists, use it. Otherwise, use `publicPath` from generated stats json.